one-ui/src/main/java/com/marklogic/hub/oneui/auth/LoginLogoutHandler.java

```diff
@@ -19,36 +19,47 @@
 import org.springframework.security.web.WebAttributes;
 import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
 import org.springframework.security.web.authentication.logout.LogoutSuccessHandler;
+import org.springframework.stereotype.Controller;
 
-import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 import java.io.IOException;
 
+@Controller
 public class LoginLogoutHandler implements AuthenticationSuccessHandler, LogoutSuccessHandler {
 
-    @Override
-    public void onAuthenticationSuccess(HttpServletRequest request, HttpServletResponse response, Authentication authentication) throws IOException {
-        ConnectionAuthenticationToken authenticationToken = (ConnectionAuthenticationToken)authentication;
+  @Override
+  public void onAuthenticationSuccess(
+      HttpServletRequest request, HttpServletResponse response, Authentication authentication)
```

Comment:
It seems your formatter may also have a fairly low limit for how long a line can be? I think 120 is the default. Again, there are a lot of changes here that are due just to formatting changes, making it more difficult to understand what the real changes are.